lib/Elastica/Aggregation/Filters.php

```diff
@@ -22,10 +22,12 @@ class Filters extends AbstractAggregation
      */
     public function addFilter(AbstractFilter $filter, $name = '')
     {
-        if (empty($name)) {
-            $filterArray[] = $filter->toArray();
+        $filterArray = array();
+
+        if (is_string($name)) {
```

Comment:
@stof Here is the fix that I applied for the current release: https://github.com/ruflin/Elastica/commit/5a08b5f66b76f2f91ecba6213a7a6b9c9225326d